systemtest/src/test/java/io/strimzi/systemtest/operators/topic/ThrottlingQuotaST.java

```diff
@@ -0,0 +1,357 @@
+
+        // List 'topicsCountBelowQuota' topics
+        KafkaAdminClient adminClientListJob = new KafkaAdminClient.Builder()
+                .withAdminName(listAdminName)
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(CLUSTER_NAME))
+                .withNamespaceName(NAMESPACE)
+                .withTopicOperation(AdminClientOperations.LIST_TOPICS.toString())
+                .withAdditionalConfig(getAdminClientConfig(kafkaUsername, 600000))
+                .build();
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        String listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(listPodName, topicNamePrefix + "-" + (topicsCountBelowQuota - 1));
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
+
+        // Delete 'topicsCountBelowQuota' topics
+        adminClientJob = adminClientJob.toBuilder()
+                .withAdminName(deleteAdminName)
+                .withTopicOperation(AdminClientOperations.DELETE_TOPICS.toString())
+                .build();
+        resourceManager.createResource(extensionContext, adminClientJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(deleteAdminName, NAMESPACE, 10);
+        String deletePodName = kubeClient().listPods("job-name", deleteAdminName).get(0).getMetadata().getName();
+        PodUtils.waitUntilMessageIsInPodLogs(deletePodName, "Successfully removed all " + topicsCountBelowQuota);
+
+        // List topics after deletion
+        resourceManager.createResource(extensionContext, adminClientListJob.defaultAdmin().build());
+        ClientUtils.waitForClientSuccess(listAdminName, NAMESPACE, 0);
+        listPodName = kubeClient().listPods("job-name", listAdminName).get(0).getMetadata().getName();
+        String afterDeletePodLogs = kubeClient().logs(listPodName);
+        assertThat(afterDeletePodLogs.contains(topicNamePrefix), is(false));
+        assertThat(afterDeletePodLogs, not(containsString(topicNamePrefix)));
+
+        JobUtils.deleteJobWithWait(NAMESPACE, createAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, listAdminName);
+        JobUtils.deleteJobWithWait(NAMESPACE, deleteAdminName);
+    }
+
+    private String getAdminClientConfig(String kafkaUsername, int timeoutMs) {
+        final String saslJaasConfigEncrypted = kubeClient().getSecret(NAMESPACE, kafkaUsername).getData().get("sasl.jaas.config");
+        final String saslJaasConfigDecrypted = new String(Base64.getDecoder().decode(saslJaasConfigEncrypted), StandardCharsets.US_ASCII);
+        return "sasl.mechanism=SCRAM-SHA-512\n" +
+                "security.protocol=" + SecurityProtocol.SASL_PLAINTEXT + "\n" +
+                "sasl.jaas.config=" + saslJaasConfigDecrypted + "\n" +
+                "request.timeout.ms=" + timeoutMs;
+    }
```

Comment:
Should this be in some utils or other admin class? We can use it in other testcases in the future.

Reply:
Well, this is only test/scram specific, but I can put it into KafkaAdminClient class and rename method a bit...

Reply:
Also, that's why it's private only...